src/util.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+function followPaginate(data) {
+  return new Promise((resolve, reject) => {
+    $.getJSON(data.url)
+    .then((response, status, xhr) => {
+      data.items = data.items.concat(response);
+      const link = xhr.getResponseHeader('Link');
+      let url = null;
+      if (link) {
+        const match = link.match(/<(.*?)>; rel="next"/);
+        url = match && match[1] ? match[1] : null;
+      }
+      resolve({ items: data.items, url: url });
+    })
+    .fail(reject);
+  });
+}
+
+function getAllItems(promise) {
+  return promise.then(followPaginate)
+  .then((data) => {
+    return data.url ? getAllItems(Promise.resolve(data), followPaginate) : data.items;
+  })
+}
+
+function createSCM(scm) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/9b7bf2b597e67365f1e9002c784237b58657e4cc/src/util.js#L27"></span>‘createSCM’ is defined but never used.</td>
    </tr>
  </tbody>
</table>